src/test/java/com/mentoring/model/User.java

```diff
@@ -60,7 +66,69 @@ private void registerUser(String username, String password, String email) {
         log.info("User registered:\n\tUsername: {}\n\tEmail: {}\n\tPassword: {}", username, email, password);
     }
 
+    public void loginUser(String userName, String password) {
```

Comment:
I would add a Step annotation: "Precondition step: Logging in with user: {userName} and password: {password})"